app/scripts/views/complete_reset_password.js

```diff
@@ -31,9 +32,15 @@ define(function (require, exports, module) {
     },
 
     events: {
+      'blur .check-password': 'onPasswordBlur',
```

Comment:
Hm I see this event `onPasswordBlur` is repeated twice in this PR with the same behaviour. can it be part of the mixin and written once, or no?